lib/pact/utils/metrics.rb

```diff
@@ -0,0 +1,62 @@
+require 'securerandom'
+require 'digest'
+require 'socket'
+require 'pact/version'
+
+module Pact
+  module Utils
+    class Metrics
+
+      def self.report_metric(event, category, action, value = 1)
+
+        if track_events?
+          Pact.configuration.output_stream.puts "WARN: Please note: we are tracking events anonymously to gather important usage statistics like Pact-Ruby version
+            and operating system. To disable tracking, set the 'pact_do_not_track' environment
+            variable to 'true'."
+
+          event = {
+            "v" => 1,
+            "t" => "event",
+            "tid" => "UA-117778936-1",
+            "cid" => calculate_cid,
+            "an" => "Pact Ruby",
+            "av" => Pact::VERSION,
+            "aid" => "pact-ruby",
+            "aip" => 1,
+            "ds" => ENV["CI"] || "unknown",
```

Comment:
` ENV["CI"]` will eq 'true'. You need this to return "ci" if  `ENV["CI"] == 'true'`.